README.md

```diff
@@ -88,6 +89,7 @@ Build the app and generate its .apk file.
 ```
 catkin build --no-jobserver
 ``` 
+or use Android Studio to build (and install).
```

Comment:
Note that building with Android Studio will not work the first time because some headers need to be generated by `catkin build`.